src/panels/config/automation/condition/ha-automation-condition.ts

```diff
@@ -334,7 +342,7 @@ export default class HaAutomationCondition extends LitElement {
 
   private _conditionChanged(ev: CustomEvent) {
     ev.stopPropagation();
-    const conditions = [...this.conditions];
+    const conditions = [...(this._pendingConditions || this.conditions)];
```

Comment:
This was a somewhat related but pre-existing bug that I found, when two child conditions fire conditionChanged in close proximity, the first change is lost, since the second update builds on top of `this.conditions` which has not been updated yet.

I added this _pendingConditions var to fix this, though it's not clear to me if that is preferrable to just writing this.conditions here instead, instead of waiting for this.conditions to be updated from the parent in response to the change.

Reply:
I'd prefer just writing to `this.conditions`, adding `_pendingConditions` makes the code harder to understand and thus easier for errors to slip in

Reply:
done, removed `_pendingConditions`